test/utils/csr/csrutils.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
```

Comment:
Code in this file is now moved to `test/builders/builder.go`. Remove this file and use the appropriate builder from there.